ReadMe.md

```diff
@@ -44,11 +44,11 @@ In order to build Kotlin distribution you need to have:
 - JDK 1.6, 1.7, 1.8 and 9
 - Setup environment variables as following:
 
-        JAVA_HOME="path to JDK 1.8"
-        JDK_16="path to JDK 1.6"
-        JDK_17="path to JDK 1.7"
-        JDK_18="path to JDK 1.8"
-        JDK_9="path to JDK 9"
+        export JAVA_HOME="path to JDK 1.8"
+        export JDK_16="path to JDK 1.6"
+        export JDK_17="path to JDK 1.7"
+        export JDK_18="path to JDK 1.8"
+        export JDK_9="path to JDK 9"
```

Comment:
I think these instructions should not be taken as a snippet ready to be copy-pasted into a shell script. This just says to declare environment variables with the following names on any platform and any shell, and since there's no `export` on Windows, having it here would be confusing for Windows users.

Reply:
OK, I get it, and I remove this commit, please see the second commit.